src/iOS.Core/Views/FormEntryTableViewCell.cs

```diff
@@ -195,5 +165,75 @@ public void Select()
                 TextField.BecomeFirstResponder();
             }
         }
+
+        public void BuildSecureCell(UIButton button)
+        {
+            button.TitleLabel.Font = UIFont.FromName("bwi-font", 28f);
+            button.SetTitle(BitwardenIcons.Eye, UIControlState.Normal);
+            button.TouchUpInside += (sender, e) =>
+            {
+                TextField.SecureTextEntry = !TextField.SecureTextEntry;
+                button.SetTitle(TextField.SecureTextEntry ? BitwardenIcons.Eye : BitwardenIcons.EyeSlash, UIControlState.Normal);
+            };
+        }
```

Comment:
Change to something like
```c#
public void ConfigureToggleSecureTextButton(bool useSecondaryButton = false)
{
    var button = useSecondaryButton ? SecondButton : Button;
    ...
```
Using it like this is less confusing for a caller to do so, given that in this clear it's clear that the button is the one already in the cell and in the other way you can pass any button as a parameter